apis/core/lib/csvw-builder/index.ts

```diff
@@ -47,6 +48,54 @@ function mappedColumn({ csvMapping, columnMapping, column }: CsvwBuildingContext
   return csvwColumn
 }
 
+async function mappedReferenceColumn({ csvMapping, columnMapping, resources }: CsvwBuildingContext & { columnMapping: ReferenceColumnMapping }): Promise<Initializer<Csvw.Column>> {
+  const csvwColumn: Initializer<Csvw.Column> = {
+    propertyUrl: csvMapping.createIdentifier(columnMapping.targetProperty).value,
+  }
+
+  const identifierMappings = columnMapping.identifierMapping
+  const columnNameMap = await identifierMappings.reduce<Promise<Map<string, string>>>(async (mapP, mapping) => {
+    const map = await mapP
```

Comment:
`mapP`?

Reply:
"P" is for "Promise". I had a lot of issues making Typescript happy with the async reduce... it's probably improvable